recipe/meta.yaml

```diff
@@ -9,7 +9,9 @@ source:
   sha256: 209823306f2cbedab6ff70997e0d236fcfd1864ca9ad082cbfdb196e7386daed
 
 build:
-  number: 0
+  number: 1
+  run_exports:
+    - {{ pin_subpackage('flatbuffers', max_pin='x.x') }}
```

Comment:
Did you mean to pin to minor version?

Reply:
Yeah... this was kinda confusing, since they moved to calendar versioning. I think pinning to major may be safe. Though historical data was limited for recent versions.

Reply:
ok.

Reply:
https://github.com/conda-forge/flatbuffers-feedstock/pull/37#issuecomment-1345339801

Thanks for the review